.gitignore

```diff
@@ -6,6 +6,13 @@ build
 # Files created for Helm dependencies
 charts/**/charts/**.tgz
 
+# Acoid checking in the license
+charts/hivemq-edge/files/*.edgelic
+charts/hivemq-edge/files/*.jks
+
+# Acoid checking in test values
+values.yaml
```

Comment:
This looks a bit fishy. Doesn't this prevent to add a new default `values.yaml` for another Helm chart?

Reply:
You are right, removed it.